### DISCUSSION TITLE
Custom Emails with variables

### DISCUSSION THREAD

**Discussion by Shoey5** [berrystephenw/OctoText#102] [Ideas]
Just a thought but what about the idea (if it's possible) for a future version to be able to generate your own formatted custom emails/texts using variables. This could allow additional information that may be relevant to individual use and allow for added variables to be added in future updates of OctoPrint/OctoText maybe even variables related to other Plugins (if that's possible)

BTW, This plugin is AWESOME! Thank you for making it!

Example: 

Subject:
%AlertStatus% %Profilename%

Email: 

User: %User%

%ProfileName% %Date% %Time%
Location: Room 1

Job: %FileName% Completed in %PrintTime%

%AlertMessage%

%Image%

**Comment by berrystephenw** [comment]
I'm sure that is possible, but that would be a lot of coding - and right now I'm enjoying my summer ;) I've settled into bug fixes and small enhancements as I have time (which is why the last RC took me so long to get out). 

Glad you enjoy the plug-in. Tell your friends :)

**Reply by Shoey5** [reply]
Something to keep in mind if you ever get into the mood to code. Will definitely be recommending it! :)
